openpype/modules/muster/muster.py

```diff
@@ -110,16 +112,10 @@ def get_auth_token(self, username, password):
         self.save_credentials(token)
 
     def save_credentials(self, token):
-        """
-        Save credentials to JSON file
-        """
-        data = {
-            'token': token
-        }
+        """Save credentials to JSON file."""
 
-        file = open(self.cred_path, 'w')
-        file.write(json.dumps(data))
-        file.close()
+        with open(self.cred_path, "w") as f:
```

Comment:
```suggestion
        with open(self.cred_path, 'w') as f:
```

I swapped the quote style to double quotes in my suggestion but it seems that this python file uses single quotes the whole time so it felt a bit mismatched. :) Maybe preserve this with single quotes then too.